### DISCUSSION TITLE
Could one endpoint provides multiple ML services? 

### DISCUSSION THREAD

**Discussion by ShelbyHero** [bentoml/BentoML#4277] [Q&A]
Hello!

I want to deploy multiple machine learning models on BentoML, but I want to deploy them to the same endpoint. 
That is, a endpoint provides multiple independent ML services. 

For example,I want to deploy model1 and model2 on the sever, model1 use the "localhost:5000/model1", and model2 use the "localhost:5000/modelA2" . 
As far as I know, I could make model1 use the "localhost:5000/model1" and make model2 use the"localhost:5001/model2 ", but that's not what I want. 

Can BentoML do this?

Thanks for the answer!

**Comment by aarnphm** [comment]
Closed this because of duplication

**Comment by simonMoisselin** [comment]
Hey any updates on this?
Thanks!

**Reply by parano** [reply]
Hi @simonMoisselin - this is always supported in BentoML and many of the example projects use this feature. for example: https://github.com/bentoml/BentoYolo/blob/main/service.py
